steering_vectors/steering_vector.py

```diff
@@ -101,6 +112,7 @@ def apply(
         operator: Optional[PatchOperator] = None,
         multiplier: float = 1.0,
         min_token_index: int = 0,
+        token_indices: List[int] | slice | Tensor | None = None,
```

Comment:
nit: also need to update the docstring with this new param so it will be reflected in API docs

Reply:
Done in [83b0ccf](https://github.com/steering-vectors/steering-vectors/pull/21/commits/83b0ccff5fa9fdedafe08830592102e55fd0b980)